<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_cie</class>
 <widget class="QDialog" name="create_cie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>358</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>414</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignTop">
    <widget class="QGroupBox" name="grp_Scaner">
     <property name="title">
      <string>Read Target</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="edt_prefix">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edt_command_line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_cmd_line">
        <property name="text">
         <string>Command Line</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_prefix">
        <property name="text">
         <string>Result Prefix</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_scan_target">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Run Scanner</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignTop">
    <widget class="QGroupBox" name="grp_ti3check">
     <property name="title">
      <string>TI3 Check</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_select_ti3s">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Select TI3s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_perpath_info">
        <property name="text">
         <string>Patches</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_overal_info">
        <property name="text">
         <string>Overview</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn_create_cie">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Create CIE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chk_is_monohrome">
        <property name="text">
         <string>Monohrome</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="QLabel" name="lbl_file_list">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>57</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
